sdk/scheduler/src/test/resources/valid-exhaustive.yml

```diff
@@ -1,7 +1,9 @@
 name: "data-store"
-principal: "data-store-principal"
-zookeeper: master.mesos:2181
-api-port: {{PORT0}}
+scheduler:
+  principal: "some-other-principal"
+  zookeeper: non-default-zk:2181
+  api-port: {{PORT_API}}
+  web-url: http://webui.data-store.mesos
```

Comment:
Done. Moving back to parent level (adjacent to service name) per offline discussion.